### PR TITLE
Fix autoloading

### PR DESCRIPTION
**Pull request by downsider**

### CODE REVIEW

**Review by PawsJa01 — APPROVED** [review]

**Review by David-Denny — APPROVED** [review]

File: src/Exception/InitialisationException.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Lexide\PuzzleDI\Exception;
+
+class InitialisationException extends \Exception
```
Comment (coolbreeze84) [inline src/Exception/InitialisationException.php:5]: PHP PER coding style say this should be written as `class InitialisationException extends \Exception {}`

**Review by coolbreeze84 — CHANGES_REQUESTED** [review]

File: src/Exception/InitialisationException.php (hunk 1 above)
Comment (downsider) [reply src/Exception/InitialisationException.php:5]: Lexide repos don't yet follow PER-2, they're still on PSR-12

Comment (coolbreeze84) [reply src/Exception/InitialisationException.php:5]: What sort of a repo is this?!?

**Review by coolbreeze84 — APPROVED** [review]

File: src/Exception/InitialisationException.php (hunk 1 above)
Comment (downsider) [reply src/Exception/InitialisationException.php:5]: Shush you!

**Pull request merged**